### DISCUSSION TITLE
" Your account has been flagged "

### DISCUSSION THREAD

**Discussion by kika1s1** [community/community#114579] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I woke up from my sleep to work on a project I had started. When I opened my GitHub, I received notifications stating, 'Your account has been flagged.' I tried contacting the GitHub support team, but they did not respond. What is going on? If anyone has experienced this issue and found a solution, I would greatly appreciate your help.

**Comment by JackTNorris** [comment, marked as answer]
@kika1s1 did you get this resolved? Have a similar thing going on

**Reply by kika1s1** [reply]
> @kika1s1 did you get this resolved? Have a similar thing going on

Yes after a month they managed to resolve my issues.

**Reply by Bi873** [reply]
[]()

**Reply by still-hash-ir** [reply]
That's unfortunate. I am facing a similar issue with my account **hash-ir**. I have followed up with them via email, started a community discussion, and even tweeted about it. But there has been no response whatsoever!

**Reply by kika1s1** [reply]
> That's unfortunate. I am facing a similar issue with my account **hash-ir**. I have followed up with them via email, started a community discussion, and even tweeted about it. But there has been no response whatsoever!

Since the team have great volume of questions regarding to this please wait patiently i think they will solve your problem soon.

**Reply by Jayantnis** [reply]
reponse @kika1s1

**Comment by Imran-imtiaz48** [comment]
I'm facing a smiliar issue. I was working on repository all of a sudden i received an email that your account has been flagged. Its been 2 week i have appeal github support team but no response.

**Comment by recluzedev1** [comment]
#2899601 I have also same issue but i generate ticket which did't get any response yet after 40 days

**Reply by rioyuda02** [reply]
Hi, 
you can follow up again, exactly
